app/views/spree/admin/orders/_bulk_actions.html.haml

```diff
@@ -3,23 +3,22 @@
     %span{ "data-controller": "checked-feedback", "data-checked-feedback-translation-value": "spree.admin.orders.index.selected" }
       = t("spree.admin.orders.index.selected", count: 0)
 
-  %div.plain.ofn-drop-down.disabled{ "data-checked-target": "disable", "data-controller": "dropdown", "data-action": "click->dropdown#toggle" }
-    %span{ class: 'icon-reorder' }
-      ="#{t('admin.actions')}".html_safe
-    %span
-      %i{ "data-dropdown-target": "arrow", "data-expanded-class": "icon-caret-up", "data-collapsed-class": "icon-caret-down" }
-
-    %div.menu{ "data-dropdown-target": "menu" }
-      %div.menu_item
-        %span.name{ "data-controller": "modal-link", "data-action": "click->modal-link#open", "data-modal-link-target-value": "resend_confirmation" }
-          = t('spree.admin.orders.index.resend_confirmation')
-      - if Spree::Config[:enable_invoices?]
+  %div.plain.ofn-drop-down.disabled{ "data-checked-target": "disable" }
```

Comment:
`±`
This `.ofn-drop-down` element is the one that has hover styles, but because of the padding, the `details` element is smaller so there's an area around the edge where clicking does nothing:
![Screenshot 2024-02-27 at 12 45 22 pm](https://github.com/openfoodfoundation/openfoodnetwork/assets/4188088/07a2c163-4110-4a78-970f-21cd3d775c06)

Maybe this element can become the `details` element?

Reply:
Aaah ..., I have only clicked on the title and so haven't notice. I added margin+paddin so that the summary element fit 100% in the details that also fits 100% in the div.
Because, the details itself is not clickable, only the summary is.

Regarding the class `ofn-drop-down` and the details merged to get one element, it is a good idea.
But the ofn-drop-down is used elsewhere (2 places) with Angular to get the same result.
May be it could be a new issue/PR to unify the drop down component to a details + css component